Source/WebCore/dom/Document.h

```diff
@@ -2343,7 +2334,7 @@ class Document
     WeakHashSet<VisibilityChangeClient> m_visibilityStateCallbackClients;
     bool m_deferResizeEventForVisibilityChange { false };
 
-    std::unique_ptr<HashMap<String, WeakPtr<Element, WeakPtrImplWithEventTargetData>, ASCIICaseInsensitiveHash>> m_accessKeyCache;
+    std::optional<HashMap<String, WeakPtr<Element, WeakPtrImplWithEventTargetData>, ASCIICaseInsensitiveHash>> m_accessKeyCache;
```

Comment:
Not sure it matters for Document (I see a distinct lack of packing above indicating size is unlikely to be all that important), but this is slightly bigger than the std::unique_ptr<> since it requires the extra bool. If it doesn't exist already, the HashMap family of types could probably be made Markable<> aware to get this back.

Reply:
I do not think it will matter for Document objects.

Reply:
I did have this exact same thought (including the `Markable` part) while working on this